flow-typed/src/ChorusClient.js

```diff
@@ -0,0 +1,12 @@
+interface ChorusClient {
+	open(url: string): Promise<Event>,
+	close(): void,
+	connect(): void,
+	publishStep(
+		pattern: string,
+		callback: (...args: Array<string>) => number | string | void,
+		technicalDescription?: string,
```

Comment:
Optional parameter I take it - probably fine but can contain something which helps identify the code which publishes the step to a developer when a step fails